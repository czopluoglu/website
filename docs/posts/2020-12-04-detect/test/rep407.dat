01111011101111111010
11111010101010001110
10000010000100110101
10001001100011011111
11100011110101101111
10000100101100011010
11001001001010000011
00001010001110111100
00010010000000000010
11111001011010110111
01100100101001010010
00001001011010100011
10000010001101101111
01010100011111101111
11010001000100100111
00001010000010011011
00101001100000000010
01000001001011001011
01101001000000010010
00101010111101101111
10000001011011111110
10111100010010100110
11001101001100111111
11000001101110000011
11110011010111100111
10111101110100100011
00110001001010101100
10111100100010100111
00001001011000101110
10000011010011000101
10000011010000010011
01110111001110110011
10100000101111011111
11000000011010111000
11011110111111011011
01100110111110110010
11110011011110011111
10011111101010111111
11011100000001111111
11110111111000010011
10111010101110100110
11001000111010000011
01101101011101111100
00000000001000111111
10001100001100101111
00110001001010111111
00001001011100100010
00011001001111000010
00011000001010100011
11010101001011100101
11000001101011111110
01001010011111100110
10100100010111110111
00010001011100011011
10111111010001011110
00000001101101000110
01001000110010000100
00000001011100001111
00110001111010101110
11110100001111011011
00101001110011111111
11000001000011000010
00011001001101110110
10101110011111111110
00000101000000101011
10010111111110110111
10001100001101101011
00011011111001101111
10110001110111101111
00000100001110111111
10101100110010110011
00100100010110000101
11000000001000000000
11111111100110110001
11011101101011010011
11110011001010101010
01001000001001100011
11110111101101111110
11110001111110101011
11011111111101100111
11010001001010101001
00101001000111100111
11010110110101100111
11011010110101101110
10111101110011101110
00000011001010000010
10111111110111101111
11111111011101110010
00001110101110101111
10001001110011110100
11001110101101000110
11010101100000100000
01111010111101011111
00110100110010100100
00011001110111110111
11001100000000100110
01110011101001000100
10100001000010010011
11111111111011110111
10110001101101111111
00001011001010000111
01000000111111101111
00100001000010101101
11100101011011101110
01001001111100111111
10101010111010111111
10111110110010001010
11011111111010101111
00000001100000100010
10100111111111111111
11001101111110000010
11110111111010101011
10100000011100100000
00001100111000110010
10001101001010100110
11011011111100101011
11011000111011001111
11111001001100001011
01010111100011000011
11000100111111101111
11110000011000111110
10010001000101100011
01100000000111011111
10101001101100110111
01001111001010001011
01001111111010111101
10001001110000000011
01010111111010100011
10011001101010011000
11010000001100101111
00000000001010101110
01000100001010100111
00000010000000000010
00011111110000110011
00110001001011111010
11101110111110111110
11111011001001111110
11001100101010010100
11010011101111111111
11110101101010001011
10001101001010010111
10101111111110111010
00011110001110000110
10011011100101111010
00001110000000101011
01110101111001000000
01011000011110101110
10101011111010100111
10001001011010100011
00100011101110111110
00000100010010101110
10010111110110011110
11101111111111011001
11111111101011101011
01001101100111101111
10111100011111101110
11110101111001000010
01001111101111110110
11001011001011100111
01101100011100100010
11111011011010111100
10110111101001110110
11111111110110010111
11010111011010100100
01001110101000111111
11110101011101100110
11000000100001111101
11111100111111101111
11011111111001110101
10100101110010110110
00001101101010110110
10111011111100110001
01011010100000101011
01110011111000101111
00001000011110110111
10101101111010001111
01111101101011100110
11111100111110011111
01010011011010000100
01001000001001110010
01001000101100101110
11001110011110010110
10011001011010111010
01010100100111101111
10001011111010111011
00000011111101101111
11000100101011110100
01000000011001100110
10111110011110100111
11010001101101101111
01010011111100110111
11101101111011011011
11101011111010100110
11111000111010111110
00010101010000000110
10001000001100111100
00011001001110001111
11101111001110111110
11010001001010100111
10110101111001110110
10000010001111111111
10000000000000001100
01001111111111111111
10100101111111101011
01100100110010000000
01000000010011101000
11111011111001100010
10110101001001110111
10110100110111111111
01101001101000010111
00010111111111100101
11100101001010011001
10111010111010010111
00001101010000000011
00011001000010010101
10011101100110111111
11000100010010111111
10111111011011111110
01100001111011011111
11001010100110111110
00101100101110110101
01000111001111101011
10000100001100101111
10101001110101111101
01111010100011100000
10101010111001000001
10100001101101001110
01000010100100100110
01000011111010100011
00000001000110100010
10100001110000100111
11110100101100011111
01011001101001100110
00010001101110110111
01010000001000000011
00000011101011011101
10010001000010000011
01111000011010000010
10000101001010111110
00000000101110010110
00100001000010100110
00010001111000000001
10010111011111101110
10110110111001001110
01010101000011000010
00110000111010101111
01100011111111101011
00100001101010111100
00110110010010100111
00110111010010100011
