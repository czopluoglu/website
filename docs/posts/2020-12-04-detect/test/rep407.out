-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep407.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.00

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.12
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7123
                   IDN index value:   0.6895
                           Ratio r:   0.5935

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    3    1    3    3    3    1    3    3
    4    4    3    2    4    4    4    4    1    1

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    4    8   19   20

   -----------CLUSTER  2-------------

    2   14

   -----------CLUSTER  3-------------

    3    5    6    7    9   10   13

   -----------CLUSTER  4-------------

   11   12   15   16   17   18
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d++-+-++++--+----++-
   +d++-+++-++---------
   ++d-+-++--++-++-----
   -+-d+---+-----+++-+-
   +-++d------+-+-+--+-
   -+---d+--+++--+-----
   +++--+d----+---+--++
   +++----d-+-+++---+--
   +--+----d-+++-+--+--
   ++---+-+-d--++------
   -++--+--+-d++++++---
   --+-+++++-+d+----+--
   +------+++++d+--+---
   --+-+--+-++-+d---+++
   --++-+--+-+---d+-++-
   ---++-+---+---+d++++
   ---+------+-+--+d-++
   +------++--+-+++-d++
   +--++-+------+++++d+
   ------+------+-++++d
 
 No cross validation for this DETECT run
